ovirt-el8-ppc64le-deps.repo.in

```diff
@@ -71,43 +71,47 @@ repo_gpgcheck=0
 enabled=1
 enabled_metadata=1
 
-[ovirt-@OVIRT_SLOT@-advanced-virtualization]
-name=Advanced Virtualization packages for $basearch
-mirrorlist=http://mirrorlist.centos.org/?arch=$basearch&release=8&repo=virt-advanced-virtualization
-enabled=1
+[ovirt-@OVIRT_SLOT@-centos-advanced-virtualization]
+name=CentOS-$releasever - Advanced Virtualization
+mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=$basearch&repo=virt-advancedvirt-common
+#baseurl=http://mirror.centos.org/$contentdir/8-stream/virt/$basearch/advancedvirt-common/
 gpgcheck=1
-gpgkey=https://www.centos.org/keys/RPM-GPG-KEY-CentOS-SIG-Virtualization
+enabled=1
 module_hotfixes=1
+gpgkey=https://www.centos.org/keys/RPM-GPG-KEY-CentOS-SIG-Virtualization
 
 [ovirt-@OVIRT_SLOT@-centos-ovirt44]
-name=CentOS-8 - oVirt 4.4
-mirrorlist=http://mirrorlist.centos.org/?arch=$basearch&release=8&repo=virt-ovirt-44
-enabled=1
+name=CentOS-$releasever - oVirt 4.4
+mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=$basearch&repo=virt-ovirt-44
```

Comment:
Why not use $stream here?

Reply:
> Why not use $stream here?

Because in CentOS Linux, Alma and Rocky $stream is not defined.